python/mlcroissant/mlcroissant/_src/core/graphs/utils.py

```diff
@@ -5,22 +5,14 @@
 import networkx as nx
 
 
-def pretty_print_graph(graph: nx.Graph, simplify=False):
+def pretty_print_graph(graph: nx.Graph):
```

Comment:
Are we sure this is not handy in some circumstances? You would obtain the same if you would remove the `simplify=True` args in the function calls below, and keep the extra simplify option here, no?

Reply:
It seems I haven't found a use case for it. Worst case it's still in the git history :)